src/share/poudriere/image.sh

```diff
@@ -503,6 +538,9 @@ tar)
 		mkminiroot
 	fi
 	;;
+zsnapshot)
+	rsync -ac --delete ${WRKDIR}/world ${WRKDIR}/mnt
```

Comment:
poudriere bundles cpdup and put it upfront in the path, so you can use it instead of rsync :)